src/frontend/src/binder/select.rs

```diff
@@ -34,25 +34,43 @@ pub struct BoundSelect {
     pub from: Option<Relation>,
     pub where_clause: Option<ExprImpl>,
     pub group_by: Vec<ExprImpl>,
-    pub schema: Schema,
+    schema: Schema,
 }
 
 impl BoundSelect {
-    /// The names returned by this [`BoundSelect`].
-    pub fn names(&self) -> Vec<String> {
-        self.aliases
+    pub fn create(
```

Comment:
Feels unnecessary to have a separate `pub create`. The only place to create `BoundXxx` is always `bind_xxx`.

Reply:
+1